### DISCUSSION TITLE
Sonoma Unable to Select Photo by Albums

### DISCUSSION THREAD

**Discussion by Anty55624** [RhetTbull/osxphotos#1316] [General]
Apple iMac (Sonoma) unable to select photos from albums when uploading photos to third party sites. Drop down menu has disappeared. Can only select photos from entire library.

**Comment by RhetTbull** [comment]
Is your question about the `osxphotos` tool? There are no drop-down menus in the tool so I'm confused about what you're asking?

**Comment by Anty55624** [comment]
Apologies… my question relates to Mac OS Sonoma 14.2 

If have inadvertently posted a question that is not relevant to this forum then please ignore. I'm new to this hub and was simply trying to seek a solution
to the problem I'm experiencing.

**Reply by RhetTbull** [reply]
Hi @Anty55624 this forum is for the `osxphotos` command line tool which works with the Apple Photos app to export your photos. It is not a forum for Apple Photos itself. If you have specific questions about the Apple Photos app, I recommend the [r/ApplePhotos](https://www.[reddit.com/r/ApplePhotos/](https://www.reddit.com/r/ApplePhotos/)) subreddit on reddit.

**Comment by Anty55624** [comment]
Thank you again. I really don’t know how to access that link or progress my problem further.
But thank you anyway
Regards Ant
